### DISCUSSION TITLE
The candle shows a line, no fluctuation, what to do?

### DISCUSSION THREAD

**Discussion by zbkmr** [klinecharts/KLineChart#364] [Q&A] [answered]
I'm developing a vue js application. I added the chart to my app. When the page loads, I get the first data from the service and then update the data with the data from the socket. But on my chart the candles look like straight lines and there is no fluctuation.

**Here you can find my data example.**
[data.txt](https://github.com/liihuu/KLineChart/files/10488016/data.txt)

https://klinecharts.com/guide/faq There is a suggestion about this problem on the page, but I have no idea how to use it.
setPriceVolumePrecision(pricePrecision, volumePrecision)

When my chart is first opened, it looks like there is ripple, but when I zoom in it becomes a straight line.
![image](https://user-images.githubusercontent.com/60570361/214255011-ae69155e-959d-4852-9715-bd37e1a8f958.png)

here is the zoomed image
![image](https://user-images.githubusercontent.com/60570361/214255308-6e41bf98-3bc7-4898-9746-e56d5d3230c6.png)

The candles should look like in this example.
https://codesandbox.io/s/aged-morning-nzi3lc?file=/src/App.vue

**Comment by liihuu** [comment, marked as answer]
```setPriceVolumePrecision(6, 6)```

**Comment by Pavee-R** [comment]
Hi @liihuu, is this only included in KLineChart 
Is that possible to use this approch with KLineChart Pro ?

**Reply by Pavee-R** [reply]
I have try set pricePercision and volumePrecision but still got the issue on KlineChartPro @liihuu
